### DISCUSSION TITLE
Can mmdeploy support a model which not belong to mmlab?

### DISCUSSION THREAD

**Discussion by zhongqiu1245** [open-mmlab/mmdeploy#2160] [Q&A] [answered]
Hello, thank you for your amazing job!
Can mmdeploy support a model which not belong to mmlab?
For example, I write a xxx-Net on my own and I want to use mmdeploy to deploy it on Android. 
(I don't want to rewrite it to make mmlab support xxx-Net becaude the work of rewriting xxx-Net is so heavy.)
I've converted xxx-Net into onnx already. Can I use mmdeploy to deploy this onnx on Android?
Thank you in advance!

**Comment by irexyc** [comment, marked as answer]
@zhongqiu1245 

For model deplement on Android, mmdeploy chose ncnn as backend. Compared to ncnn, mmdeploy may provide a simple api that hide the pre/post process.

Have you used mmdeploy on android before? If so, you may convert the onnx to ncnn and edit `deploy.json` and `pipeline.json` manually. If not, I think it's better to use ncnn directly.

**Reply by zhongqiu1245** [reply]
thank you!
